src/components/MoneyRequestHeaderStatusBar.js

```diff
@@ -1,24 +1,35 @@
 import React from 'react';
 import {View} from 'react-native';
+import PropTypes from 'prop-types';
 import styles from '../styles/styles';
-import useLocalize from '../hooks/useLocalize';
 import Text from './Text';
 
-function MoneyRequestHeaderStatusBar() {
-    const {translate} = useLocalize();
+const propTypes = {
+    /** Title displayed in badge */
+    title: PropTypes.string.isRequired,
 
+    /** Banner Description */
+    description: PropTypes.string.isRequired,
+
+    /** Whether we show the border bottom */
+    shouldShowBorderBottom: PropTypes.bool.isRequired,
+};
+
+function MoneyRequestHeaderStatusBar({title, description, shouldShowBorderBottom}) {
+    const borderBottomStyle = shouldShowBorderBottom ? styles.borderBottom : {};
     return (
-        <View style={[styles.dFlex, styles.flexRow, styles.alignItemsCenter, styles.overflowHidden, styles.ph5, styles.pb3, styles.borderBottom, styles.w100]}>
+        <View style={[styles.dFlex, styles.flexRow, styles.alignItemsCenter, styles.flexGrow1, styles.overflowHidden, styles.ph5, styles.pb3, borderBottomStyle]}>
```

Comment:
I noticed there are no screenshots for mobile platforms, and I think we'd need to ensure there are no visual regressions introduced in this PR